libraries/ui-library/src/components/six-datepicker/six-datepicker.tsx

```diff
@@ -736,6 +740,8 @@ export class SixDatepicker {
           label={this.label}
           required={this.required}
           error-text={this.errorText}
```

Comment:
Any particular reason to use kebab case here, while the other attributes use camelCase?

Reply:
Autocomplete! Fixed now